packages/dappkit/tsconfig.json

```diff
@@ -5,15 +5,14 @@
     "baseUrl": ".",
     "declaration": true,
     "jsx": "preserve",
-    "lib": ["dom", "es2015", "es2016"],
+    "lib": ["dom", "es2015", "es2016", "es2017", "es2018", "esnext"],
```

Comment:
This is assuming that this will be always be running in an RN environment?

Reply:
Yeah, i don't see dappkit much useful outside?